examples/deployment/scripts/2-gold-asc.js

```diff
@@ -0,0 +1,23 @@
+const {
+  printAssetHolding,
+  transferAsset,
+  asaOptIn,
+  asaOptInContract,
+  transferMicroAlgosContract
+} = require('../src/asa-helpers');
+
+async function run(runtimeEnv, accounts, deployer) {
+
+  const goldOwnerAccount = deployer.accountsByName.get("gold-owner-account");
+  const johnAccount = deployer.accountsByName.get("john-account");
+
+  if (deployer.isDeployMode) {
```

Comment:
I didn't do this check for ASA since we only use this script in deploy mode.
I think we don't need to do it here too.

Reply:
ok